### PR TITLE
gltf-model-next

### PR DESCRIPTION
**Pull request by 7dir**
fix readme

### CODE REVIEW

File: src/loaders/README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Unsure whether to use `json-model` or `object-model`? Check the console for erro
 ## Usage
 
 ```html
-<a-entity json-model="src: url(my-model.json);">
-</a-entity>
+<a-entity json-model="src: url(my-model.json);"></a-entity>
+<a-entity gltf-model-next="src: url(https://raw.githubusercontent.com/mrdoob/three.js/master/examples/models/gltf/RiggedSimple/glTF/RiggedSimple.gltf);"></a-entity>
 ```
```
Comment (donmccurdy) [inline src/loaders/README.md]: Good call! For brevity, let's just use a short (fake) model name here:

```html
<a-entity gltf-model-next="src: url(my-model.gltf);"></a-entity>
```

**Comment by 7dir** [comment]
done

**Comment by donmccurdy** [comment]
Thanks!

**Comment by 7dir** [comment]
Could I add example with full features model?

**Comment by donmccurdy** [comment]
I think it's probably not necessary — this component will be in A-Frame core with the next release, replacing `gltf-model`, and then it will disappear from aframe-extras.

**Pull request merged**